src/components/Table/components/Body/Body.module.scss

```diff
@@ -9,6 +9,10 @@
     border-right: 1px solid $color-7;
   }
 
+  &--firstRowStyled {
```

Comment:
weźmy tu konsekwentnie uzywajmy notacji z myślnikiem first-row-styled :D

Reply:
u góry jest externalBorder, więc używałem camelCase'a
<img width="426" alt="Screenshot 2023-08-07 at 09 28 55" src="https://github.com/landingi/ui-kit/assets/132905788/2eedcf50-3091-4dce-aefb-fdf49fb3d1d2">